actions/linting/action.yml

```diff
@@ -0,0 +1,59 @@
+name: Linting
+description: Runs pre-commit over the repo
+
+inputs:
+  ansible-roles-config:
+    description: The ansible requirements.yml file
+    default: missing
+
+  pre-commit-config:
+    description: The pre-commit config file of the current repo
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout source
+      uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
```

Comment:
```suggestion
      uses: actions/checkout@v4
```

Reply:
This is actually suggested my `renovate` for security reasons - think it did it automatically